govuk_chat_evaluation/rag_answers/data_models.py

```diff
@@ -1,6 +1,29 @@
 from deepeval.test_case import LLMTestCase
-from pydantic import BaseModel
+from pydantic import BaseModel, model_validator
+from enum import Enum
+from typing import Any, Optional
+import uuid
 
+from deepeval.metrics import (
+    FaithfulnessMetric,
+    AnswerRelevancyMetric,
+    BiasMetric,
+)
+from deepeval.models import DeepEvalBaseLLM
+
+class GeneratedCaseParams(Enum):
+    QUESTION = "question"
+    LLM_ANSWER = "llm_answer"
+    RETRIEVED_CONTEXT = "retrieved_context"
```

Comment:
This doesn't seem to be used anywhere?

Reply:
No, you are right, I think it was from your initial implementation of the GEval class.